src/Dofs/ConstraintHandler.jl

```diff
@@ -191,17 +191,22 @@ function _add!(ch::ConstraintHandler, dbc::Dirichlet, bcfaces::Set{Tuple{Int,Int
 
     # loop over all the faces in the set and add the global dofs to `constrained_dofs`
     constrained_dofs = Int[]
+	redundant_faces = NTuple{2, Int}[]
```

Comment:
Configure your editor to use 4 spaces instead of tabs :)
```suggestion
    redundant_faces = NTuple{2, Int}[]
```
(same below)